ql/instruments/asianoption.cpp

```diff
@@ -21,6 +21,7 @@
 #include <ql/instruments/asianoption.hpp>
 #include <ql/time/date.hpp>
 #include <utility>
+#include <ql/settings.hpp>
```

Comment:
```suggestion
#include <ql/settings.hpp>
#include <utility>
```
(not strictly needed, but usually we try to keep headers in some sort of order for ease of reading)